README.md

```diff
@@ -39,6 +39,25 @@
     - 50002: Shippable marketing ui
     - 50003: Shippable admin panel
 
+### Load balancers
+- The usual(and recommended) way of exposing Shippable service end points is
+  via loadbalancers. The LB's are not required to be exposed to the public
+  facing internet but should be accessible to all users in a VPN. A typical
+  setup creates following routing
+
+  User (shippable.mycompany.com) -> Route 53 -> internal LB -> Shippable API
```

Comment:
Could you capitalize "internal"?  It stands out because everything else on the line starts with a capital lettter.